app/models/signed_up_team.rb

```diff
@@ -81,4 +81,20 @@ def self.topic_id_by_team_id(team_id)
       signed_up_teams.first.topic_id
     end
   end
+
+  # Remove a specific signed_up_teams record for a given topic and team.
+  def self.drop_off_signup_record(topic_id,team_id)
```

Comment:
Why not "remove_signup_record"?  Also, I'm a little uncomfortable with a class method here, although I grant it might be the cleanest way to implement.